README.md

```diff
@@ -1,407 +1,436 @@
-    shorten_path = true -- currently the default value is true
-  }
+```lua 
+use {
+  'nvim-lua/telescope.nvim',
+  requires = {{'nvim-lua/popup.nvim'}, {'nvim-lua/plenary.nvim'}}
 }
 ```
 
-### Default Configuration Keys
-
+#### Usage
 
-- ( Missing configuration description for many items here, but I'm trying :smile: )
-- `file_ignore_patterns`:
-    - List of strings that are Lua patterns that, if any are matched, will make result be ignored.
-    - Please note, these are Lua patterns. See: [Lua Patterns](https://www.lua.org/pil/20.2.html)
-    - Example:
-        - `file_ignore_patterns = { "scratch/.*", "%.env" }`
-        - This will ignore anything in `scratch/` folders and any files named `.env`
+Try the command `:Telescope find_files<cr>`
+  to see if `telescope.nvim` is installed correctly.
 
-## Examples
+```viml
+" Find files using Telescope command-line sugar.
+nnoremap <leader>ff <cmd>Telescope find_files<cr>
+nnoremap <leader>fg <cmd>Telescope live_grep<cr>
+nnoremap <leader>fb <cmd>Telescope buffers<cr>
+nnoremap <leader>fh <cmd>Telescope help_tags<cr>
 
-```vim
-nnoremap <Leader>p <cmd>lua require'telescope.builtin'.git_files{}<CR>
+" Using lua functions
+nnoremap <leader>ff <cmd>lua require('telescope.builtin').find_files()<cr>
+nnoremap <leader>fg <cmd>lua require('telescope.builtin').live_grep()<cr>
+nnoremap <leader>fb <cmd>lua require('telescope.builtin').buffers()<cr>
+nnoremap <leader>ff <cmd>lua require('telescope.builtin').help_tags()<cr>
 ```
 
-Searches over files in a git folder. Note: This does not work outside a git repo folder.
+See [built-in pickers](#built-in-pickers)
+  for the list of all built-in functions.
 
-```vim
-nnoremap <Leader>p <cmd>lua require'telescope.builtin'.find_files{}<CR>
-```
 
-Search over files in your `cwd` current working directory.
+<!-- Section -->
 
-```vim
-nnoremap <silent> gr <cmd>lua require'telescope.builtin'.lsp_references{}<CR>
-```
-
-Search over variable references from your Language Server.
-
-```vim
-nnoremap <Leader>en <cmd>lua require'telescope.builtin'.find_files{ cwd = "~/.config/nvim/" }<CR>
-```
-
-Find all the files in your nvim config.
 
-### Available keys for `defaults`
+## Customization
 
-- `generic_sorter`:
-    - Description: The sorter to be used for generic searches.
-    - `default`: `require('telescope.sorters').get_generic_fuzzy_sorter
-- `file_sorter`:
-    - Description: The sorter to be used for file based searches.
-    - `default`: `require('telescope.sorters').get_fuzzy_file
+You can configure `telescope.nvim` at two levels:
+  globally, through the `setup()` method,
+  or in a finer-grained manner
+  by passing options to individual built-in pickers.
 
-### Full Example
+|  [Presentation](#presentation) | [Sorting](#sorting) | [Mappings](#mappings)
 
-```vim
-lua <<EOF
--- totally optional to use setup
-require('telescope').setup {
-  defaults = {
-    shorten_path = false -- currently the default value is true
-  }
-}
-EOF
+#### Presentation
```

Comment:
There is also a `scroll_strategy` which can be `cycle`. See
https://github.com/nvim-telescope/telescope.nvim/blob/24057365dae52e76d6f2afb67f99eb94b51b40fc/lua/telescope/pickers.lua#L672-L688

Reply:
Thanks @Conni2461 , I've added it